libs/design/src/core/theming/components/theme-switch/theme-switch.module.ts

```diff
@@ -4,19 +4,19 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 
 import { DaffButtonModule } from '@daffodil/design';
 
-import { DaffodilThemeSwitchComponent } from './theme-switch.component';
```

Comment:
This component/module doesn't belong in `@daffodil/design`.

Reply:
you mean it should be a subpackage or its own root level package?

Reply:
Sorry for the vagueness, I meant that it's not really on us to pre-fabricate a component for users that just drops into their UI. I'd much prefer that they have control over that.

Reply:
I moved it into the daffio because it's not really a branding component.